protocols/872-cosmosid-ngs-library-prep-part1/nexteraXT_dna_library_prep_part1.ot2.py

```diff
@@ -0,0 +1,104 @@
+from opentrons import labware, instruments, robot
+
+"""
+Tagment Genomic DNA
+Amplify Libraries
+"""
+# labware setup
+gDNA_plate = labware.load('96-PCR-flat', '1')
+out_plate = labware.load('96-PCR-flat', '2')
+# out_plate = labware.load('biorad-hardshell-96-PCR', '2')
+tuberack = labware.load('tube-rack-2ml', '5')
+
+# reagent setup
+atm = tuberack.wells('A1')  # Amplicon Tagment Mix
+td = tuberack.wells('B1')  # Tagment DNA Buffer
+nt = tuberack.wells('C1')  # Neutralize Tagment Buffer
+npm = tuberack.wells('D1')  # Nextera PCR Master Mix
+index_1 = tuberack.wells('A2', to='D4')  # Index 1 (i7) adapters
+index_2 = tuberack.wells('A5', to='D6')  # Index 2 (i5) adapters
+
+tipracks50 = [labware.load('tiprack-200ul', slot) for slot in ['3', '4']]
+tipracks10 = [labware.load('tiprack-10ul', slot) for slot in ['6', '7']]
+
+# pipette setup
+p50 = instruments.P50_Single(
+    mount='left',
+    tip_racks=tipracks50)
+
+p10 = instruments.P10_Single(
+    mount='right',
+    tip_racks=tipracks10)
+
+
+def run_custom_protocol(
+        number_of_samples: int=24):
+
+    # define sample locations
+    samples = gDNA_plate.wells()[:number_of_samples]
+
+    index1 = {name: [] for name in range(len(index_1))}
+    index2 = {name: [] for name in range(len(index_2))}
+
+    if number_of_samples <= 24:
+        output = [well
+                  for col in gDNA_plate.cols('1', to='6')
+                  for well in col.wells('A', to='D')][:number_of_samples]
+        for index, well in enumerate(output):
+            index1[index // 4].append(well)
+            index2[index % 4].append(well)
+    else:
+        output = [well for well in out_plate.wells()][:number_of_samples]
+        for index, well in enumerate(output):
+            index1[index // 8].append(well)
+            index2[index % 8].append(well)
+
+    """
+    Tagment genomic DNA
+    """
+    # Add Tagment DNA Buffer to each well
+    p50.distribute(10, td, [well.top() for well in samples])
+
+    # Add normalized gDNA to each well
+    p10.transfer(5, samples, output, new_tip='always')
+
+    robot.comment("Centrifuge at 280 × g at 20°C for 1 minute. Place on the \
```

Comment:
This doesn't do what you think it does. Use this:

robot.pause(///text here///)